build-ps/debian/extra/mysql-systemd-start

```diff
@@ -48,13 +38,6 @@ sanity () {
 		chmod 770 ${MYSQLFILES}
 	fi
 
-	if [ ! -d ${MYSQLKEYRING} -a ! -L ${MYSQLKEYRING} ];
```

Comment:
Not sure why we got rid of this or are we doing this somewhere else.
This is creating a keyring folder needed for InnoDB Tablespace encryption functionality.